types/uppercamelcase/index.d.ts

```diff
@@ -1,8 +1,8 @@
-// Type definitions for uppercamelcase
+// Type definitions for uppercamelcase 3.0
 // Project: https://github.com/samverschueren/uppercamelcase
 // Definitions by: York Yao <https://github.com/plantain-00>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-declare function upperCamelCase(...args: string[]): string;
 export = upperCamelCase;
-declare namespace upperCamelCase { }
```

Comment:
For code:
```ts
import upperCamelCase = require("uppercamelcase");
console.log(upperCamelCase("foo"));
```

compiled with tsconfig.json:
```js
{
  "compilerOptions": {
      "target": "es5",
      "module": "esnext",
      "moduleResolution": "node"
  }
}
```
the generated js file is:
```js
console.log(upperCamelCase("foo"));
```
You see, there is no `var upperCamelCase = require("uppercamelcase");` in generated js file.

Why I use `"module": "esnext"` rather than `"module": "commonjs"`?
Because I use dynamic import(https://github.com/Microsoft/TypeScript/pull/14774#issuecomment-306072693).
Why I use `"moduleResolution": "node"`?
Because I use angular(https://angular.io/guide/typescript-configuration#tsconfigjson), removing it will generate: `Could not resolve module @angular/core`.

So, how to make `import foo = require()` syntax work with `angular` and `dynamic import`?